pkg/common/cns-lib/volume/manager.go

```diff
@@ -174,6 +174,16 @@ type CreateVolumeExtraParams struct {
 	IsPodVMOnStretchSupervisorFSSEnabled bool
 }
 
+// CreateSnapshotExtraParams consist of values required by the CreateSnapshot interface and
+// are not present in the CNS CreateSnapshot spec.
+type CreateSnapshotExtraParams struct {
+	StorageClassName           string
+	StoragePolicyID            string
+	Namespace                  string
+	Capacity                   *resource.Quantity
+	IsStorageQuotaM2FSSEnabled bool
```

Comment:
I see that IsStorageQuotaM2FSSEnabled is added to ExtraParams. What if it was enabled before the snapshot was created but was disabled after the snapshot was created?

Reply:
I think there will be no impact on further snapshot create requests, but snapshot quota will left allocated to previously create snapshots (with FSS enabled). we may need to add some logic to revoke such quota.